src/pretix/base/models/memberships.py

```diff
@@ -163,8 +163,12 @@ def __str__(self):
     def attendee_name(self):
         return build_name(self.attendee_name_parts, fallback_scheme=lambda: self.customer.organizer.settings.name_scheme)
 
-    def is_valid(self, ev=None):
-        if ev:
+    def is_valid(self, ev=None, valid_from=None, valid_from_not_chosen=False):
+        if valid_from_not_chosen:
+            return not self.canceled and self.date_end >= now()
+        elif valid_from:
+            dt = valid_from
```

Comment:
...um es ein bisschen von der event und membership validity abzugrenzen:
```suggestion
    def is_valid(self, ev=None, ticket_valid_from=None, valid_from_not_chosen=False):
        if valid_from_not_chosen:
            return not self.canceled and self.date_end >= now()
        elif ticket_valid_from:
            dt = ticket_valid_from
```

am liebsten würde ich `valid_from_not_chosen` auch noch in (`ticket_`)`valid_from_not_chosen_yet` umbenennen, aber dann müssten man es eigentlich überall ändern... ich habe eine weile gebraucht um zu verstehen, dass das nur ein temporärer zustand ist.